praw/models/reddit/comment.py

```diff
@@ -208,7 +208,15 @@ class CommentModeration(ThingModerationMixin):
 
     .. code:: python
 
-       reddit.comment(id="dxocde6").mod.approve()
+       praw.models.reddit.comment.CommentModeration(id='dxocde6').approve()
```

Comment:
PRAW is designed in such a way that its users will never have to directly instantiate any class besides `Reddit`. In line with that, I don't think it's appropriate to document instantiating classes directly in the documentation. Please remove this code sample and edit the sentence below to "This class can be accessed through the ``.mod`` attribute of moderatable comments:".

I like how you separated the example out to two lines. I think it makes it look more readable.